### DISCUSSION TITLE
nodejs is buggy in version 16.9.1!

### DISCUSSION THREAD

**Discussion by ThiagoSilm** [termux/termux-packages#7531] [General]
it stopped working, just when i updated it doesn't run anything anymore, i've tried everything but with no success. termux version 0.0.115

**Comment by Yonle** [comment]
![Screenshot_2021_0915_075511](https://user-images.githubusercontent.com/63401712/133353277-58c139e0-263c-4ced-bb2a-27a1ce4097d0.png)
Guess, i can reproduce the bug.

**Comment by Yonle** [comment]
But because node is not returning any error, You may fix your code.
![Screenshot_2021_0915_075938](https://user-images.githubusercontent.com/63401712/133353355-5a0c8604-4959-4a2c-a531-18e4ae7834e1.png)
> P.S. "console.log(\\"a\\")" = console.log("a")

**Comment by Yonle** [comment]
By the way, #7534
